connection.go

```diff
@@ -21,14 +20,14 @@ type Conn struct {
 	// after Ready is closed.
 	MountError error
 
-	// File handle for kernel communication. Only safe to access if
-	// rio or wio is held.
+	// File handle for kernel communication.
 	dev *os.File
-	wio sync.RWMutex
```

Comment:
I like getting rid of this and making it up to the user, higher in the stack, to lock, but I'm not sure we do right now....